### PR TITLE
Add custom error response

### PR DESCRIPTION
**Pull request by mcsky**
In some situation, you want to be able to return error manually.

### CODE REVIEW

File: src/Bridge/Symfony/Response/ErrorResponse.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        $this->errors = $errors;
+        $this->errorKey = $errorKey;
+    }
+
+    /**
+     * @return UserDataError[]
+     */
+    public function getErrors(): array
+    {
+        $errors = new UserDataError($this->errorKey, []);
```
Comment (Nek-) [inline src/Bridge/Symfony/Response/ErrorResponse.php]: Should be singular. Also I think it's better to create it in the constructor and store an array of `UserDataError`. (it allows us to add a method `addError()` in the future)

Also the attribute `errors` could be protected to allow extension.

**Review by Nek- — CHANGES_REQUESTED** [review]
Thank you very much. This contribution is nice & complete.

I think some improvements can be done here, what do you think?

**Comment by mcsky** [comment]
With pleasure :)

If you mean your review comment, the changes are done. I prefer to keep this class simple for the moment and add more things in the future if needed.

File: src/Bridge/Symfony/Response/ErrorResponse.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+use Biig\Melodiia\Response\AbstractUserDataErrorResponse;
+use Biig\Melodiia\Response\Model\UserDataError;
+
+class ErrorResponse extends AbstractUserDataErrorResponse
+{
+    /** @var UserDataError[] */
+    protected $errors;
+
+    /** @var string */
+    private $errorKey;
```
Comment (Nek-) [inline src/Bridge/Symfony/Response/ErrorResponse.php:14]: Oops, can you remove this useless field?

**Pull request merged**